custom_components/adaptive_lighting/manifest.json

```diff
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/basnijholt/adaptive-lighting#readme",
   "iot_class": "calculated",
   "issue_tracker": "https://github.com/basnijholt/adaptive-lighting/issues",
-  "requirements": ["ulid-transform"],
```

Comment:
This is still required for Home Assistant ≤ 2023.03

Reply:
BTW maybe this warrants a patch release?

Reply:
Yeah, these are the only changes: https://github.com/basnijholt/adaptive-lighting/compare/1.10.0...main

All fixes :)

Reply:
When did `fix_rgb_color_temp_swaps` get merged in? I thought that didn't make it into 1.10.0

Reply:
Oh, that is true! But it would be somewhat of in indication to the users to not install 1.10.0 but only 1.10.1.

I can also cherry-pick all fixes only?

Reply:
Should we go with 1.10.1? I've drafted using [1.11.0](https://github.com/basnijholt/adaptive-lighting/releases/tag/untagged-348ba258f023da1892c5)

Reply:
see https://github.com/basnijholt/adaptive-lighting/releases/tag/untagged-348ba258f023da1892c5

Reply:
I figured 1.11.0 would get the bad taste from 1.10.0 out of everyone's mouth

Reply:
I just released [1.11.0](https://github.com/basnijholt/adaptive-lighting/releases/tag/1.11.0) please edit anything you do not like

Reply:
We are not introducing any new features, right?

Typically minor patch releases are for features, and patch are for fixes.

I don't like having a broken release tagged without having a fix.

Reply:
I saw you tagged a release on a commit on this branch. That should never be done because we will delete this branch after merging. Then this commit will be "dangling".

Reply:
I did not include any features, we only merged fixes

Reply:
But you seem to really like 1.11.0. Go ahead, I assume you will deal with the folks complaining that 1.10 doesn't work anyway :joy:

Reply:
Maybe you should do it as I'm a bit new to the ebb and flow of how these usually go.

Reply:
Uhhh second thought maybe rename it to 1.10.1